src/test/java/io/jenkins/plugins/jfrog/configuration/JFrogPlatformBuilderTest.java

```diff
@@ -21,4 +21,16 @@ public void testIsUnsafe() {
         assertTrue(JFrogPlatformBuilder.isUnsafe(false, "http://acme.jfrog.io"));
         assertTrue(JFrogPlatformBuilder.isUnsafe(false, "https://acme.jfrog.io", "http://acme.jfrog.io"));
     }
+
+    @Test
+    public void testIsEmptyOrStartingWithProtocol() {
+        assertFalse(JFrogPlatformBuilder.isEmptyOrStartingWithProtocol(""));
```

Comment:
According to the method's outputs, its name should be changed - 

isEmptyOrStartingWithProtocol --> isIInvalidProtocolOrEmptyUrl